CONTRIBUTING.md

```diff
@@ -51,3 +58,10 @@ The format can be described more formally as follows:
 ```
 <package>: <subpackage>, <what changed>. [Fixes #<issue-number>]
 ```
+
+[discord server]: https://discord.gg/6CG7M6wF
```

Comment:
```suggestion
[discord server]: https://discord.gg/8hrxYEVPE5
```

Reply:
Good catch @AriehSchneier, thanks for that. :bow: PTAL